sktime/classifiers/tests/test_proximity.py

```diff
@@ -13,520 +14,508 @@ def run_classifier_on_dataset(classifier, dataset_loader,
     classifier.fit(X_train, y_train)
     predict_probas = classifier.predict_proba(X_test)
     testing.assert_array_equal(predict_probas, expected_predict_probas)
-    predictions = classifier.predict(X_test)
-    testing.assert_array_equal(predictions, expected_predictions)
+    # predictions = classifier.predict(X_test)
+    # testing.assert_array_equal(predictions, expected_predictions)
 
 
-def test_proximity_forest_on_gunpoint():
-    classifier = ProximityForest(debug = True, random_state = 0, num_trees = 10, num_stump_evaluations = 5)
-    run_classifier_on_dataset(classifier, load_gunpoint, forest_gunpoint_predict_probas, forest_gunpoint_predictions)
+def test_proximity_stump_on_gunpoint():
+    classifier = ProximityStump(random_state=0)
+    run_classifier_on_dataset(classifier, load_gunpoint, stump_gunpoint_predict_probas, stump_gunpoint_predictions)
 
 
 def test_proximity_tree_on_gunpoint():
-    classifier = ProximityTree(debug = True, random_state = 0, num_stump_evaluations = 5)
+    classifier = ProximityTree(random_state=0, find_stump=proximity.best_of_n_stumps(5))
     run_classifier_on_dataset(classifier, load_gunpoint, tree_gunpoint_predict_probas, tree_gunpoint_predictions)
 
-
-def test_proximity_stump_on_gunpoint():
-    classifier = ProximityStump(debug = True, random_state = 0)
-    run_classifier_on_dataset(classifier, load_gunpoint, stump_gunpoint_predict_probas, stump_gunpoint_predictions)
-
+def test_proximity_forest_on_gunpoint():
+    classifier = ProximityForest(random_state=0, n_trees=3, find_stump=proximity.best_of_n_stumps(1))
+    run_classifier_on_dataset(classifier, load_gunpoint, forest_gunpoint_predict_probas, forest_gunpoint_predictions)
 
 stump_gunpoint_predict_probas = np.array([
```

Comment:
perhaps only run a few samples of gunpoint?

Reply:
The tests made quite a problem for me. Currently, they test PF / PT / PS well, but take a while. If I trim the dataset, I risk making the problem too simple and not thoroughly testing the classifiers, though the tests would be quicker.

I don't know which way is both, what's your thoughts @mloning @jasonlines @TonyBagnall ?

Reply:
I prefer quicker for unit tests, we do test them for equivalence against the java code base etc

Reply:
yes trim, the tests are not for the quality of the algorithm, just that it builds ok